set.mm

```diff
@@ -89689,6 +89713,59 @@ Axiom of Power Sets (unlike ~ ensymb ).  (Contributed by BTernaryTau,
     $( $j usage 'ensymfib' avoids 'ax-pow'; $)
   $}
 
+  ${
+    $d A f g $.  $d B f g $.  $d C f g $.
+    $( Transitivity of equinumerosity for finite sets, proved without using the
+       Axiom of Power Sets (unlike ~ entr ).  (Contributed by BTernaryTau,
+       10-Sep-2024.) $)
+    entrfil $p |- ( ( A e. Fin /\ A ~~ B /\ B ~~ C ) -> A ~~ C ) $=
+      ( vf vg cen wbr cfn wcel cv wf1o wex bren wa exdistrv 19.42vv ccom ancoms
+      f1oco f1oenfi sylan2 exlimivv sylbir sylan2br 3impb syl3an2b syl3an3b ) B
+      CFGAHIZABFGZBCDJZKZDLZACFGZBCDMUIUHABEJZKZELZULUMABEMUHUPULUMUPULNUHUOUKN
+      ZDLELZUMUOUKEDOUHURNUHUQNZDLELUMUHUQEDPUSUMEDUQUHACUJUNQZKZUMUKUOVAABCUJU
+      NSRACUTTUAUBUCUDUEUFUG $.
+    $( $j usage 'entrfil' avoids 'ax-pow'; $)
+  $}
+
+  ${
+    $d A x $.  $d B x $.
+    $( A set equinumerous to a finite set is finite.  (Contributed by Mario
+       Carneiro, 12-Mar-2015.)  Avoid ~ ax-pow .  (Revised by BTernaryTau,
+       23-Sep-2024.) $)
+    enfii $p |- ( ( B e. Fin /\ A ~~ B ) -> A e. Fin ) $=
+      ( vx cen wbr cfn wcel wa com wrex wex isfi df-rex sylbb ensymfib biimparc
+      cv 19.41v w3a sylibr simp1 nnfi biimpar 3adant3 entrfil syld3an2 3ad2ant1
+      wb mpbid syl3an1 jca 3expa eximi sylbir syl2an2 ancoms ) ABDEZBFGZAFGZUQU
+      RHZACQZDEZCIJZUSUTVAIGZVBHZCKZVCURVDBVADEZHZCKZUQBADEZVFURVGCIJVICBLVGCIM
+      NURVJUQBAOPVIVJHVHVJHZCKVFVHVJCRVKVECVDVGVJVEVDVGVJSVDVBVDVGVJUAVDVAFGZVG
+      VJVBVAUBVLVGVJSVAADEZVBVLVABDEZVGVJVMVLVGVNVJVLVNVGVABOUCUDVABAUEUFVLVGVM
+      VBUHVJVAAOUGUIUJUKULUMUNUOVBCIMTCALTUP $.
+    $( $j usage 'enfii' avoids 'ax-pow'; $)
+  $}
+
+  $( Equinumerous sets have the same finiteness.  (Contributed by NM,
+     22-Aug-2008.)  Avoid ~ ax-pow .  (Revised by BTernaryTau, 23-Sep-2024.) $)
+  enfi $p |- ( A ~~ B -> ( A e. Fin <-> B e. Fin ) ) $=
+    ( cen wbr cfn wcel wa ensymfib pm5.32i enfii sylbi expcom impbid ) ABCDZAEF
+    ZBEFZONPONGOBACDZGPONQABHIBAJKLPNOABJLM $.
+  $( $j usage 'enfi' avoids 'ax-pow'; $)
+
+  $( Transitivity of equinumerosity for finite sets, proved without using the
+     Axiom of Power Sets (unlike ~ entr ).  (Contributed by BTernaryTau,
+     23-Sep-2024.) $)
+  entrfi $p |- ( ( B e. Fin /\ A ~~ B /\ B ~~ C ) -> A ~~ C ) $=
+    ( cfn wcel cen wbr enfii 3adant3 entrfil syld3an1 ) ADEZABFGZBDEZBCFGZACFGN
+    MLOABHIABCJK $.
+  $( $j usage 'entrfi' avoids 'ax-pow'; $)
+
+  $( Transitivity of equinumerosity for finite sets, proved without using the
+     Axiom of Power Sets (unlike ~ entr ).  (Contributed by BTernaryTau,
+     23-Sep-2024.) $)
+  entrfir $p |- ( ( C e. Fin /\ A ~~ B /\ B ~~ C ) -> A ~~ C ) $=
```

Comment:
Is this one really useful, since you have already proved symmetry ?

Reply:
I personally feel like transitivity is an important enough result that it is worth having all 3, but yes, you could use entrfil along with ensymfib or entrfi along with enfi, depending on what fits best.